prutsen/prutsen.pde

```diff
@@ -3,20 +3,20 @@ import ddf.minim.*;
 import ddf.minim.ugens.*;
 
 Serial port;                                                                     // Data received from the serial port      
-int buffer = 0;                                                                  // Load the buffer 
+float buffer = 0;                                                                // Load the buffer 
 float volume = 0.0;                                                              // Creating volume var 0.0 as starting point
 float maxGain = 2;                                                               // Setting a Maximum on volume
 float minGain = -100;                                                            // Setting a Minimum on volume
-int BPM = 0;                                                                     // Creating BPM var 0 as starting point
+float BPM = 0;                                                                   // Creating BPM var 0 as starting point
 float maxBPM = 180;                                                              // Setting a Maximun on BPM rate
 float audioSampleBPM = 100;                                                      // Setting the standard samplerate to 100 BPM
-int x = 100;
-int y = 450;
-int z = 650;
-int w = 450;
-float r_volume = 0;
-float r_BPM = 0;
-float max_rotation_speed = 100;
+float x = 100;                                                                   // Creating float X with value 100
+float y = 450;                                                                   // Creating flpat Y with vlua 450
+float z = 650;                                                                   // Creating float Z with calue 650
+float w = 450;                                                                   // Creating float W with value 450
+float r_volume = 0;                                                              // Creating float r_volume with value 0
+float r_BPM = 0;                                                                 // Creating float r_BPM with value 0
```

Comment:
je kunt misschien beter comments maken als 'x position of rotating volume bar' en 'rotation angle (degrees) of volume bar'